wiki/Tournaments/Official_support/en.md

```diff
@@ -167,6 +167,9 @@ Profile badge images must adhere to the following standards:
 - It must clearly display the logo, motif, or name of your tournament.
 - It must be cleanly designed and of reasonable visual quality.
 - It must not include any sort of sponsorship or promote anything besides the tournament.
+- It must not include any assets which were not specifically made for the tournament and adheres to the [Content usage guidelines](/wiki/Rules/Content_usage_permissions).
+  - This does not include assets that are open-source or public-use (i.e. fonts, icons, etc).
```

Comment:
i dont think this should be written as a sub-bullet because it isn't really related. you could also have assets that aren't meant for public use but were made for the tournament for example. also, fonts and icons don't generally fall into this category so idk why they are used to explain this

looking at the next sub-bullet as well, I feel like all of this is kind of unnecessary to write when you've already linked to content usage guidelines, which gets the same points across

Reply:
content usage guidelines do not cover paid font usage or licensing which these are meant to cover it is very barebones.